docs/releasing.md

```diff
@@ -1,34 +1,16 @@
 Releasing
 =========
 
-We intend to automate the releasing process and move it to the CI systems but at the following manual steps are required:
+Releasing is automated a script `src/scripts/releasing.sh`
```

Comment:
"…automated via a script…"

Reply:
Done.
I've changed the policy that new commits dismisses **all** (there is nothing more fine grain) old approvals, cause I got tired to ask for re-approvals.